src/iso9660.rs

```diff
@@ -221,7 +221,7 @@ pub struct File {
     pub length: u32,
 }
 
-#[derive(Debug, Serialize, Deserialize, Clone, Copy)]
+#[derive(Debug, Serialize, Deserialize, Clone, Copy, PartialEq)]
```

Comment:
Nit: might as well derive `Eq` in both places while we're here.

Reply:
Done!